be/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif ()
 
 # Set boost/stacktrace use backtrace api to unwind
 add_definitions(-DBOOST_STACKTRACE_USE_BACKTRACE)
-add_definitions(-DPRINT_ALL_ERR_STATUS_STACKTRACE)
+#add_definitions(-DPRINT_ALL_ERR_STATUS_STACKTRACE)
```

Comment:
why comment out this config？

Reply:
If this marco is enabled, BE will print all error status to log, it may cause performance problem. So that I disabled it by default. Should enable it manually if it is needed.